storage/bootstrap/bootstrapper/commitlog/source.go

```diff
@@ -348,7 +350,7 @@ func (s *commitLogSource) mergeShard(
 			if shardResult == nil {
 				shardResult = result.NewShardResult(len(unmergedShard.encodersBySeries), s.opts.ResultOptions())
 			}
-			shardResult.AddSeries(unmergedBlocks.id, nil, seriesBlocks) // FOLLOWUP(prateek): include tags in commit log reader
+			shardResult.AddSeries(unmergedBlocks.id, unmergedBlocks.tags, seriesBlocks)
```

Comment:
Nice, perhaps add a similar bootstrapper integration test for the commit log similar to the fs one?

Reply:
Discussed in person, gonna hold off and add one for the index in the next P.R